Source/WebCore/accessibility/mac/AXObjectCacheMac.mm

```diff
@@ -399,12 +399,33 @@ static void exerciseIsIgnored(AccessibilityObject& object)
 
 void AXObjectCache::onDocumentRenderTreeCreation(const Document& document)
 {
-    if (m_sortedIDListsInitialized) {
-        // We only need to do this work if the sorted ID list has already been initialized.
-        m_deferredDocumentAddedList.add(document);
-        if (!m_performCacheUpdateTimer.isActive())
-            m_performCacheUpdateTimer.startOneShot(0_s);
-    }
+    if (!m_sortedIDListsInitialized)
+        return;
+    // We only need to do this work if the sorted ID list has already been initialized.
+
+    RefPtr object = getOrCreate(document.renderView());
+    if (!object || !object->isWebArea())
+        return;
+    queueUnsortedObject(object.releaseNonNull(), PreSortedObjectType::WebArea);
+}
+
+void AXObjectCache::deferSortForLiveRegion(Ref<AccessibilityObject>&& object)
+{
+    if (!m_sortedIDListsInitialized)
```

Comment:
Can you just move the m_sortedIDListsInitialized check into queueUnsortedObject?

Reply:
Yes, done it latest commit!